source/admin_manual/list_of_events.rst

```diff
@@ -812,9 +812,17 @@ IMAP command
 |                     |                                                      |
 |                     | .. versionadded:: v2.3.9                             |
 +---------------------+------------------------------------------------------+
-| cmd_name            | IMAP command name (e.g. ``FETCH``)                   |
+| cmd_name            | IMAP command name uppercased (e.g. ``FETCH``).       |
 |                     |                                                      |
 |                     | .. versionadded:: v2.3.9                             |
+|                     |                                                      |
+|                     | .. versionchanged:: v2.3.11 Contains ``unknown`` for |
+|                     |                     unknown command names.           |
++---------------------+------------------------------------------------------+
+| cmd_input_name      | IMAP command name exactly as sent (e.g. ``fetcH``).  |
+|                     | Also contains the name of unknown commands.          |
```

Comment:
I'd reword this a little:

IMAP command name exactly as sent (e.g., ``fetcH``) regardless of whether or not it is a valid.

Reply:
I removed "a", i.e. "... it is valid." I was wondering whether to change to "it is a valid command." but maybe that's repeating the command too much.